core/pom.xml

```diff
@@ -173,6 +173,12 @@
 		<dependency>
 			<groupId>org.apache.commons</groupId>
 			<artifactId>commons-digester3</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>cglib</groupId>
+					<artifactId>cglib</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
Does not appear to be needed for our use of the digester, and good to get rid of it since it is not JDK17 compatible.